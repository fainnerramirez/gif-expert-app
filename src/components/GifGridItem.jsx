import React from "react";
import PropsTypes from "prop-types";

const GifGridItem = ({ title, url }) => {
  return (
    <div className="col animate__animated animate__fadeInUp">
      <div className="card" style={{ width: "17rem" }}>
        <img src={url} className="card-img-top" alt={title} />
        <div className="card-body bg-warning">
          <h5 className="card-title text-center">{title}</h5>
        </div>
      </div>
    </div>
  );
};

GifGridItem.propTypes = {
  title: PropsTypes.string.isRequired,
  url: PropsTypes.string.isRequired,
};

export default GifGridItem;
