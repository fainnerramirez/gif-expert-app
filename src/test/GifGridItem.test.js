import { shallow } from "enzyme";
import GifGridItem from "../components/GifGridItem";

describe("Pruebas en el archivo gifGridItem.js", () => {
  test("Se debe mostrar el componente correctamente", () => {
    const view = shallow(<GifGridItem />);

    expect(view).toMatchSnapshot();
  });
});
