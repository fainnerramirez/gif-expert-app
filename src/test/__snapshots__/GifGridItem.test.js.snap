// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el archivo gifGridItem.js Se debe mostrar el componente correctamente 1`] = `
<div
  className="col animate__animated animate__fadeInUp"
>
  <div
    className="card"
    style={
      Object {
        "width": "18rem",
      }
    }
  >
    <img
      className="card-img-top"
    />
    <div
      className="card-body"
    >
      <h5
        className="card-title"
      />
    </div>
  </div>
</div>
`;
